rn/Teacher/i18n/locales/index.js

```diff
@@ -36,6 +36,7 @@ module.exports = {
   fr: require('./fr.json'),
   'fr-CA': require('./fr-CA.json'),
   ht: require('./ht.json'),
+  is: require('./is.json'),
```

Comment:
Oh I forgot about this. We need to do this anytime a new language is added?

Reply:
yeah, every time.